dotnet/src/Extensions/Planning.ActionPlanner/skprompt.txt

```diff
@@ -8,6 +8,4 @@ For each function the list includes details about the input parameters.
 - List of functions:
 {{this.ListOfFunctions}}
 - End list of functions.
-Goal: {{ $input }}
-{"plan":{
```

Comment:
I would undo this change:
* the JSON was semi open on purpose to reduce the chance of models introducing things that are not json
* less tokens to generate (speed concern)

Reply:
I was getting more instances of invalid JSON without it - let's chat about this today.

Reply:
Per our offline discussion, we'll do it this way for now and start bringing up test cases for other models.